analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyCollectionsShouldNotBeEnumerated.cs

```diff
@@ -520,57 +520,6 @@ public void UnknownExtensionMethods()
         list.Clear();                                       // Compliant
     }
 
-    public void WellKnownExtensionMethods()
-    {
-        var list = new List<int>();
-        list.All(x => true);                                // FN
```

Comment:
Only the first invocation was raised, then the state was reset and the others would have been Compliant